HoneyBadgers.Logic/FileDataReader.cs

```diff
@@ -36,6 +36,7 @@ private static List<T> LoadData<T>(string filePath)
             catch (Exception e)
             {
                 Console.WriteLine($"An error has occurred while reading Users records. Ex: {e.Message}");
+                //TODO: jak pominąć console.write by dostać uniwersalną informacje o błędzie?
```

Comment:
nie rozumiem tego todo? chcesz to do jakiegoś pliku zapisywać?

Reply:
Console.WriteLine wyświetli komunikat w consoli, ale classa jest związana z logiką (będzie wykorzystana w aplikacji web i dalej), a tam nie będziemy widzieć komunikatu, nie będzie przecież widoku konsoli :?